docs/application/native/guides/internationalization/i18n.md

```diff
@@ -869,9 +878,67 @@ using a Number format for the PluralFormat locale.
 
+<a name="date_interval"></a>
+## Usages of Date Intervals
+
+### Create Date Interval Object
+
+The DateInterval API (in [mobile](../../api/mobile/latest/group__CAPI__BASE__UTILS__I18N__DATE__INTERVAL__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__BASE__UTILS__I18N__DATE__INTERVAL__MODULE.html) applications) allows you to [create date intervals](#date_interval_examples).
+
+DateInterval module represents time period between pair of `i18n_udate` values. For example, this type contains
+number of milliseconds from 1 January 1970 00:00 UTC to desired date.
+
+### Format Date Intervals
+
+The DateIntervalFormat API (in [mobile](../../api/mobile/latest/group__CAPI__BASE__UTILS__I18N__DATE__INTERVAL__FORMAT__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__BASE__UTILS__I18N__DATE__INTERVAL__FORMAT__MODULE.html) applications) allows you to [format created date intervals](#date_interval_examples).
+
+Purpose of this module is to enable formatting date intervals in a language-independent manner. DateIntervalFormat module formats a date interval into text as compactly as possible.
+For example, the date interval format from "Jan 11, 2008" to "Jan 18, 2008" is
+"Jan 11-18, 2008" for English.
+
+There is no structural information in date time patterns. For any punctuation or string literals in a date time pattern,
+you do not know whether it is just a separator, a prefix, or a suffix. Without such information,
+it is difficult to generate a sub-pattern (or super-pattern) using algorithm. Therefore, formatting a date interval is pattern-driven.
+
+A skeleton:
+- Only keeps the field pattern letter and ignores all other parts in a pattern, such as space,
+  punctuations, and string literals.
+- Hides the order of fields.
+- Can hide a field's pattern letter length.
+
+For the non-digit calendar fields, the pattern letter length is important. The pattern length are MMM, MMMM, and MMMMM; EEE and EEEE.
+The field's pattern letter length is effectively considered. For example, the EEE pattern matches the short names of the weekdays
+(For instance, Mon, Tue, and so on.). If the pattern should match the entire weekday name, use the EEEE pattern.
+
+For interval formatting, ICU supports the calendar fields such as
+year, month, date, day-of-week, am or pm, hour, hour-of-day, minute, and second
+(currently, ICU does not have specific interval format date for skeletons with seconds).
+These calendar fields are defined in the following order:
+**year > month > date > hour (in day) > minute > second.**
+
+For example, the largest different calendar fields between "Jan 10, 2007" and "Feb 20,
+2008" is the **year**.
+
+For other calendar fields, the compact interval formatting is not supported. The function returns formatted date intervals depending on the interval value.
+The date interval format is {date0} - {date1}. For example, 20 - 25 September or 20 April - 15 May
```

Comment:
Please add a period (.) at the end.
For example, 20 - 25 September or 20 April - 15 May.